src/OpenTelemetry.Exporter.OpenTelemetryProtocol/Implementation/LogRecordExtensions.cs

```diff
@@ -76,17 +76,26 @@ internal static OtlpLogs.LogRecord ToOtlpLog(this LogRecord logRecord)
 
                 // TODO: Add logRecord.CategoryName as an attribute
 
+                bool bodyPopulatedFromFormattedMessage = false;
                 if (logRecord.FormattedMessage != null)
                 {
                     otlpLogRecord.Body = new OtlpCommon.AnyValue { StringValue = logRecord.FormattedMessage };
+                    bodyPopulatedFromFormattedMessage = true;
                 }
 
                 if (logRecord.StateValues != null)
                 {
                     foreach (var stateValue in logRecord.StateValues)
                     {
-                        var otlpAttribute = stateValue.ToOtlpAttribute();
-                        otlpLogRecord.Attributes.Add(otlpAttribute);
+                        if (stateValue.Key.Equals("{OriginalFormat}") && !bodyPopulatedFromFormattedMessage)
```

Comment:
A comment here referencing this is the behavior when using the extension methods will help my memory 6 months from now 😄